core/templates/dev/head/filters.js

```diff
@@ -441,3 +441,15 @@ oppia.filter('removeDuplicatesInArray', [function() {
     });
   };
 }]);
+
+oppia.filter('stripFormatting', [function() {
+  return function(html, whitelistedImgClasses) {
+    // Strip out anything between and including <>,
+    // unless it is an img whose class includes one of the whitelisted classes.
+    var regex = new RegExp(
+      '(?!<img.*class=".*(' + whitelistedImgClasses.join('|') +
```

Comment:
I checked with Ben, I think he means using something like `class="..."' explicitly, but I don't think that will work because there may be more than one class.

It seems reasonable to add closing quotes though -- I guess change ).*> to ).*".*> ?  But I think it's worth getting this in so I'll suggest it be done in a follow up PR.